apps/endatix-hub/features/public-form/ui/survey-component.tsx

```diff
@@ -58,7 +60,57 @@ export default function SurveyComponent({ definition, formId, submission }: Surv
     });
   }, []);
 
+  const interceptImages = useCallback((sender: SurveyModel, options: ValueChangingEvent) => {
```

Comment:
Have you considered renaming the `options` parameter to `event`? Since it's an event, event might be a clearer name and make it easier to understand the data being passed. Let me know what you think

Reply:
Thanks. I am aware that the "options" is standard naming convention for this parameter across events. However, since this represents our custom logic built on top of it, my recommendation was to ensure the code better aligns with our context

I don’t consider this a blocking issue for the PR, so I’m happy to leave it as is for now.